content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -20,12 +20,40 @@ Watch the following video for an introduction to the `db seed` command:
 
 </TopBlock>
 
-## Seeding your database with TypeScript
+## Requirements for seeding with TypeScript or JavaScript
 
-If you are using TypeScript, follow these steps:
+Prisma's integrated seeding functionality expects the following:
+
+1. A `prisma/seed.ts` or `prisma/seed.js` file.
```

Comment:
Is this path absolute or relative to your schema file location?

Reply:
Path is resolved to "next to schema.prisma" file

Reply:
Thanks for the clarification @Jolg42 - all mentions of this should be updated then (also below).